editor.planx.uk/src/components/ImagePreview.tsx

```diff
@@ -0,0 +1,8 @@
+import { useFileUrl, UseFileUrlProps } from "@planx/components/shared/hooks";
+import React from "react";
+
+export default function ImagePreview(props: UseFileUrlProps) {
+  const { fileUrl } = useFileUrl(props);
+
+  return <img src={fileUrl} alt="" />;
```

Comment:
nit: I think empty `alt` here is probably an accessibility violation. i wonder if there's a simple standard template we coud use like `Preview of uploaded file: ${fileUrl}` ?